app/controllers/api/v1/invoice_items_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::InvoiceItemsController < ApplicationController
+
+  def index
+    respond_with InvoiceItem.all
+  end
+
+  def show
+    respond_with InvoiceItem.find(invoice_items_params[:id])
+  end
```

Comment:
again here i don't think we need to access the id from the strong params, since here we are just finding it from the params in the path